app/jobs/notify_webhook_job.rb

```diff
@@ -18,7 +18,10 @@ def perform(notification_id)
       client = get_client(subscription)
       response = client.post(subscription.callback_url, data, 'PECS-SIGNATURE': hmac, 'PECS-NOTIFICATION-ID': notification_id)
 
-      raise "non-success status received from #{subscription.callback_url} (#{response.status} #{response.reason_phrase})" unless response.success?
+      unless response.success?
+        parsed_response = JSON.parse(response.body)
```

Comment:
I don't think we can assume that every response is guaranteed to be JSON

Reply:
maybe check the response content type and parse as json if it is json.  Actually, thinking about this more, maybe it is safer just to send through the un-parsed response as a string in the error message.

Keep in mind that suppliers are running different code / different architectures and the responses from each are likely to be different.

Reply:
Or, even better: if there is a `JSON.parse(response.body)['message']` use that, otherwise just the unparsed `response.body`

Reply:
good point, I think maybe we should just include the `response.body` as you suggest without parsing, as we'll have to handle JSON parsing errors otherwise.

Reply:
👍